src/niscope/metadata/functions_addon.py

```diff
@@ -270,14 +272,52 @@
             },
             {
                 'direction': 'in',
+                'default_value': None,
                 'name': 'numSamples',
                 'type': 'ViInt32',
                 'documentation': {
                     'description': 'The maximum number of samples to fetch for each waveform. If the acquisition finishes with fewer points than requested, some devices return partial data if the acquisition finished, was aborted, or a timeout of 0 was used. If it fails to complete within the timeout period, the function throws an exception.',
                 },
             },
             {
-                'direction': 'out',
+                'direction': 'in',
+                'enum': 'FetchRelativeTo',
+                'default_value': None,
+                'name': 'fetchRelativeTo',
+                'type': 'ViInt32',
+                'documentation': {
+                    'description': 'Position to start fetching within one record. If not set, use value of NISCOPE_ATTR_FETCH_RELATIVE_TO',
```

Comment:
NISCOPE_ATTR_FETCH_RELATIVE_TO didn't get converted to python_name by the generator.

Reply:
Yeah, I need to figure out why. I'll keep looking into it.

Reply:
See #749